tests/integration/steps/integration_steps.go

```diff
@@ -0,0 +1,622 @@
+package steps
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	resty "github.com/go-resty/resty/v2"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const RETRY_TIMEOUT = 15 * time.Minute
+const RETRY_INTERVAL = 30 * time.Second
+
+var namespace string
+var repairId string
+var releaseName string
```

Comment:
We should not have any global variables if we want to allow proper parallel execution. These should be hidden behind a struct.

Reply:
I've removed all global variables and context objects.
We either have constants or variables that are local to each test scenario.